app/views/videos/index.html.haml

```diff
@@ -6,5 +6,5 @@
       - category.recent_videos.each do |video|
         .video.col-sm-2
           = link_to video, nil do
-            = image_tag video.small_cover_url 
+            = image_tag "#{video.small_cover_url}" 
```

Comment:
It was OK before, why did you put it to interpolation? This step is unnecessary, you can delete the quotes and interpolation.